cmd/src/campaign_plans_create_from_patches.go

```diff
@@ -0,0 +1,90 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/mattn/go-isatty"
+	"github.com/pkg/errors"
+)
+
+func init() {
+	usage := `
+Create a campaign plan from a set of patches (in unified diff format) to repository branches.
+
+Stdin is expected to be a JSON array of {repository: string, baseRevision: string, patch: string}. The repository value is the repository's GraphQL ID (which you can look up given a repository name using 'src repos get -name=...').
+
+Examples:
+
+  Create a campaign plan from my.patch applied to a repository's master branch:
+
+		$ patch=$(jq -M -R -s . < my.patch)
+		$ repo=$(src repo get -name=github.com/sourcegraph/src-cli -f '{{.ID|json}}')
+		$ echo '[{"repository": $repo, "baseRevision": "master", "patch": $patch}]' |
+    	  src campaign plans create-from-patches
```

Comment:
```suggestion
    	  src campaigns plan create-from-patches
```

I actually like `plans` more than `plan`, though. But it's `plan` in the other PRs too.